tests/test_ocs_basic_install.py

```diff
@@ -114,9 +124,13 @@ def run_cmd(cmd, **kwargs):
 
     """
     log.info(f"Executing command: {cmd}")
-    r = subprocess.run(cmd.split(), stdout=sys.stdout, stderr=sys.stderr, **kwargs)
+    r = subprocess.run(
```

Comment:
It fits in single line right and is following the guidelines?

Reply:
it was 84 chars